### PR TITLE
spektral 1.1.0

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)
- [x] #14990

### CODE REVIEW

File: easybuild/easyconfigs/s/spektral/spektral-1.1.0-foss-2021b-CUDA-11.4.1.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# John Dey <jfdey@fredhutch.org>
+#
+# Fred Hutchinson Cancer Center - Seattle Washington - US
+#
+easyblock = 'PythonPackage'
+
+name = 'spektral'
+version = '1.1.0'
+local_cuda_suffix = '-CUDA-%(cudaver)s'
+versionsuffix = local_cuda_suffix
```
Comment (jfgrimm) [inline easybuild/easyconfigs/s/spektral/spektral-1.1.0-foss-2021b-CUDA-11.4.1.eb:12]: I would just simplify this to:
```suggestion
versionsuffix = '-CUDA-%(cudaver)s'
```

File: easybuild/easyconfigs/s/spektral/spektral-1.1.0-foss-2021b-CUDA-11.4.1.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('lxml', '4.6.3'),
+    ('SciPy-bundle', '2021.10'),
+    ('networkx', '2.6.3'),
+    ('tqdm', '4.62.3'),
+    ('scikit-learn', '1.0.1'),
+    ('CUDA', '11.4.1', '', True),
+    ('TensorFlow', '2.7.1', '-CUDA-%(cudaver)s'),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/s/spektral/spektral-1.1.0-foss-2021b-CUDA-11.4.1.eb:31]:
```suggestion
    ('TensorFlow', '2.7.1', versionsuffix),
```

**Comment by fizwit** [comment]
Dependency is in dev branch
EasyBuildError: 'Missing dependencies: TensorFlow/2.7.1-foss-2021b-CUDA-11.4.1 
--from-pr #14990

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in total)
himem02.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/db5f2e5fe1c2d87e1d40e9277244b58e for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@fizwit I've been a bit cheeky and committed the change to `SYSTEM`.
However, I don't appear to be able to run `--sync-pr-with-develop`.
@jfgrimm we would be keen to see this merged, because there's a request for `DeepCell`.

**Comment by fizwit** [comment]
Thanks for committing this change to spectral.
```
    ('CUDA', '11.4.1', '', SYSTEM),
```

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 1 out of 3 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/a453cfa4ac6b1f43b53e175287c6bc9d for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/a32219175f7751bcdc7ebe64eb0794fc for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Going in, thanks @fizwit!

**Comment by verdurin** [comment]
Going in, thanks @fizwit!

**Comment by verdurin** [comment]
Going in, thanks @fizwit!

**Comment by verdurin** [comment]
Going in, thanks @fizwit!

**Pull request merged**